### DISCUSSION TITLE
nickname-prefix only for changed characters, not if it's just colors

### DISCUSSION THREAD

**Discussion by NotToBlame** [EssentialsX/Essentials#5254] [Ideas and Feature Suggestions]
Hey I would like to request that `nickname-prefix` option will only actually take effect if the characters of a name change so for example (NotToBlame -> N0tToBlam3) would activate it but just adding color codes such as "&3NotToBlame" would not

Attempting to make a better way to see when players have nicknames by showing their original name next to their nickname. To help with impersonation so that people still know who is who when using a nickname. When it's only colors that change it looks a little silly to have it

**Comment by ChiboYen** [comment]
This is a good feature that I think should be added, it would allow servers to let players change the color of their own username but not allow impersonation as easily still.
